Core/Systems/AltUseUISystem.cs

```diff
@@ -19,8 +19,11 @@ public override void Load()
 
 	public override void Unload()
 	{
-		AltBar.Dispose();
-		AltBar = null;
+		Main.RunOnMainThread(() =>
+		{
+			AltBar.Dispose();
```

Comment:
Important to note: I don't know if we have to manually `Dispose` at all. If we don't, we should be able to either null it out or ignore it entirely, making this whole thing useless. I'm not sure though, I'd have to look into it more; this is still being merged for now.